http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -22,6 +22,8 @@ tags:
     description: Notebooks API. Deprecated in favor of Proxy API.
   - name: service_info
     description: Service Info API
+  - name: apps
+    description: App API info
```

Comment:
Maybe "Experiment Apps API"?

Reply:
Also while we're in here, maybe `s/Experimental Persistent Disks API/Persistent Disks API` above?

Reply:
_Also_ while we're in here we should maybe update the Proxy section to include the `/proxy/google/v1/apps/{project}/{appName}/{serviceName}` route